test/test_linalg.py

```diff
@@ -4682,6 +4683,33 @@ def test_corner_cases_of_cublasltmatmul(self, device, dtype):
         m2 = torch.randn(16, 131071, device=device).to(dtype)
         torch.nn.functional.linear(m1, m2, M)
 
+    @skipCUDAIfNotRocm
+    @dtypes(*floating_types_and(torch.bfloat16, torch.half))
+    def test_hipblaslt_corner_cases_rocm(self, device, dtype):
+        # enable hipblaslt path via env variable.
+        import os
+        os.environ["DISABLE_ADDMM_HIP_LT"] = "0"
+        if (dtype != torch.double):
```

Comment:
IMO if the intent to skip the testing for double types, it would be better to say something like:
```python
   if type == torch.double:
     raise unittest.skipTest("Not yet supported for doubles")

Reply:
Thanks, this makes sense. Let me update it.